docs/beamrunner/release/1.2.0/troubleshoot.md

```diff
@@ -0,0 +1,45 @@
+
+## Enable tracing
+Tracing can give information about the translation and execution of your Beam application and can help debug problems.
+
+### Trace levels
+Streams Runner tracing levels follow SLF4J's levels of `ERROR`, `WARN`, `INFO`, `DEBUG`, and `TRACE` with increasing verbosity and information.
+
+### Translation-time
+Streams Runner uses Beam utilities and the [streamsx.topology](http://ibmstreams.github.io/streamsx.topology/) toolkit to translate your Beam application into a Streams application, submit the application to your IBM Streams instance or IBM Streaming Analytics service, and provide connections to the instance or service for monitoring or cancellation. Log messages about these activites are printed to the console where the Beam program is executed. When specifying the program command, you can use the `--traceTranslation` pipeline option to specify a trace level for the runner, topology toolkit, and/or global scopes. For example, a
+user can debug a SAB compilation (done by the topology toolkit) without verbose Beam or Streams Runner translation logging by specifying `--traceTranslation=INFO,Streams=DEBUG`.
+
+By default, Streams Runner uses `INFO` level tracing when translating so users can view helpful messages about their application preparation and submission while the topology toolkit and Beam loggers are set to `WARN`. To set the runner, topology, or global (for Beam and its dependencies) trace levels, see the description of `traceTranslation` in [Streams Runner pipeline options](../reference/#streams-runner-pipeline-options).
+
+### Runtime
+When your application is successfully submitted and started on a Streams environment, the application will use the tracing levels set by the `--traceRuntime` and log all messages including standard output and error to log files on the distributed environment. The application trace level can be changed after the job is running from Streams Console for individual operators (transforms), processing elements (PEs), or the entire job. For more information about viewing, downloading application logs, or changing trace levels from Streams Console, see the [Troubleshooting section of the Streaming Analytics Development Guide](https://developer.ibm.com/streamsdev/docs/bluemix-streaming-analytics-development-guide/#troubleshooting).
+
+Streams Runner runtime component can be set separately from the global trace levels similar to translation-time. By default, the runtime trace level is set to `WARN` and for more information see `traceRuntime` in [Streams Runner pipeline options](../reference/#streams-runner-pipeline-options). **Note**: increasing runtime trace levels may negatively impact performance of an application.
+
+For a complete IBM Streams troubleshooting guide, see [Troubleshooting and support](https://www.ibm.com/support/knowledgecenter/en/SSCRJU_4.2.0/com.ibm.streams.pd.doc/doc/ts_parent.html).

```

Comment:
This should refer to the 4.3.0 KC (change the xref link). The KC for 4.3.0 is not active yet, but you can verify the link on our internal KC.
http://leto.svl.ibm.com:9343/help/topic/com.ibm.streams.pd.doc/doc/ts_parent.html